### ISSUE TITLE
The fault of an activity isnt showing up in the viewer (send signal) 

### ISSUE THREAD

**Issue by matt4446** [elsa-workflows/elsa-core#1381]
Here is the end of the workflow. The Send Signal Activity Faulted: 

![image](https://user-images.githubusercontent.com/1550939/129054196-c2d89a71-f409-4f55-810a-b80c0a353c2e.png)

But there isn't anything to tell me on the viewer - what, where and why it died? 

I have a handy retry dialog ready to tell me its last output, so i know the data is there: 
```
{
  "Exception": {
    "Type": "Elsa.Exceptions.CannotSetActivityPropertyValueException",
    "Message": "An exception was thrown whilst setting 'SendSignal.Input'. See the inner exception for further details.",
    "StackTrace": "   at Elsa.Services.ActivityPropertyProviders.SetActivityPropertiesAsync(IActivity activity, ActivityExecutionContext activityExecutionContext, CancellationToken cancellationToken)\r\n   at Elsa.Services.Workflows.ActivityActivator.ActivateActivityAsync(ActivityExecutionContext context, Type type, CancellationToken cancellationToken)\r\n   at Elsa.Providers.ActivityTypes.TypeBasedActivityProvider.<>c__DisplayClass6_0.<<CreateActivityTypeAsync>b__0>d.MoveNext()\r\n--- End of stack trace from previous location ---\r\n   at Elsa.Services.Workflows.WorkflowRunner.RunCoreAsync(WorkflowExecutionContext workflowExecutionContext, ActivityOperation activityOperation, CancellationToken cancellationToken)\r\n   at Elsa.Services.Workflows.WorkflowRunner.RunAsync(WorkflowExecutionContext workflowExecutionContext, ActivityOperation activityOperation, CancellationToken cancellationToken)",
    "InnerException": {
      "Type": "Jint.Runtime.JavaScriptException",
      "Message": "Cannot read property 'AccessToUtc' of undefined",
      "StackTrace": "   at <anonymous>:2:73",
      "InnerException": null
    }
  },
  "Message": "An exception was thrown whilst setting 'SendSignal.Input'. See the inner exception for further details.",
  "FaultedActivityId": null,
  "ActivityInput": null,
  "Resuming": false
}

```

Was this part of the tidy up of inputs and outputs (different providers)?

**Comment by sfmskywalker** [comment]
@matt4446 I indeed believe too that this got broken during the input/output storage providers handiwork. Will fix.

**Comment by sfmskywalker** [comment]
Fixed with 7cb8e3edd1f3b08b406f1972a0d92e2564ff4617
